src/components/calculator.js

```diff
@@ -1,75 +1,98 @@
-          <button type="button" className="signs">
+          <button type="button" onClick={this.keyClick} className="signs">
             ÷
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             7
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             8
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             9
           </button>
-          <button type="button" className="signs">
-            X
+          <button type="button" onClick={this.keyClick} className="signs">
+            x
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             4
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             5
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             6
           </button>
-          <button type="button" className="signs">
+          <button type="button" onClick={this.keyClick} className="signs">
             -
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             1
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             2
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             3
           </button>
-          <button type="button" className="signs">
+          <button type="button" onClick={this.keyClick} className="signs">
             +
           </button>
-          <button type="button" className="greyKey extra1">
+          <button
+            type="button"
+            onClick={this.keyClick}
+            className="greyKey extra1"
+          >
             {' '}
           </button>
-          <button type="button" className="greyKey extra2">
+          <button
+            type="button"
+            onClick={this.keyClick}
+            className="greyKey extra2"
+          >
             0
           </button>
-          <button type="button" className="greyKey">
+          <button type="button" onClick={this.keyClick} className="greyKey">
             .
           </button>
-          <button type="button" className="signs">
+          <button type="button" onClick={this.keyClick} className="signs">
             =
           </button>
```

Comment:
- [ ] One of the requirements for this project is to use `useState` and `props`. You have used `useState` but not props. For you to use `props`, you need to create a component and make it a child of the `Calculator` component. I would advise you to make a `Button` component and use it as a child component of the `Calculator` component. Remember that components should reflect the separation of concerns and should be reusable. 👍 😎